Build/Laptop/EMail.cc

```diff
@@ -1916,7 +1917,7 @@ static void HandleIMPCharProfileResultsMessage(void)
 		// have to place players name into string for first record
 		if (i == 0)
 		{
-			wchar_t zTemp[512];
+			wchar_t zTemp[NAME_LENGTH];
```

Comment:
There are two things here. You're talking about zName potentially overflowing somewhere else, which is not something this commit addresses (nor should it imo). The other thing is that the format specifier for swprintf contains a space, so the effective size will be one less for a fully packed (but not overflown) zName. So zTemp should be at least NAME_LENGTH+1 for the last char not to become null. Right?